scripts/run-benchmark.ps1

```
@@ -0,0 +1,176 @@
+}
+
+function Reset-RepoDir {
+    param([string]$PathToReset)
+
+    $resolvedPath = Assert-WithinRepo -PathToCheck $PathToReset
+    if (Test-Path -LiteralPath $resolvedPath) {
+        Remove-Item -LiteralPath $resolvedPath -Recurse -Force
+    }
+}
+
+function Stop-ListenerOnPort {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-ListenerOnPort' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)